### PR TITLE
Added saner production rolling log file options

### PR DESCRIPTION
**Pull request by eladamitpxi**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -23,6 +23,10 @@ log_dir: /var/log/zookeeper
 zookeeper_dir: /opt/zookeeper-{{zookeeper_version}}
 zookeeper_tarball_dir: /opt/src
 
+# Rolling file appender setttings
+rolling_log_file_max_size: 10MB
```
Comment (ernestas-poskus) [inline defaults/main.yml]: prefix variables with ```zookeeper_```

File: tasks/common-config.yml [hunk 2]
```diff
@@ -6,6 +6,12 @@
     - Restart zookeeper
   when: zookeeper_env is defined and zookeeper_env|length > 0
 
+- name: update the log4j config with saner production values
```
Comment (ernestas-poskus) [inline tasks/common-config.yml]: ```update``` uppercase -> ```Update```

File: templates/log4j.properties.j2 [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# Define some default values that can be overridden by system properties
+zookeeper.root.logger=INFO, CONSOLE, ROLLINGFILE
+zookeeper.console.threshold=INFO
+zookeeper.log.dir=.
+zookeeper.log.file=zookeeper.log
+zookeeper.log.threshold=DEBUG
+zookeeper.tracelog.dir=.
```
Comment (ernestas-poskus) [inline templates/log4j.properties.j2]: I assume that it takes configuration of log directory from https://github.com/AnsibleShipyard/ansible-zookeeper/blob/master/templates/zoo.cfg.j2#L3 ?

File: templates/log4j.properties.j2 [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Define some default values that can be overridden by system properties
+zookeeper.root.logger=INFO, CONSOLE, ROLLINGFILE
+zookeeper.console.threshold=INFO
+zookeeper.log.dir=.
```
Comment (ernestas-poskus) [inline templates/log4j.properties.j2]: I assume that it takes configuration of log directory from https://github.com/AnsibleShipyard/ansible-zookeeper/blob/master/templates/zoo.cfg.j2#L3 ?

File: defaults/main.yml (hunk 1 above)
Comment (eladamitpxi) [inline defaults/main.yml]: fixed

File: tasks/common-config.yml (hunk 2 above)
Comment (eladamitpxi) [inline tasks/common-config.yml]: fixed

File: templates/log4j.properties.j2 (hunk 3 above)
Comment (eladamitpxi) [inline templates/log4j.properties.j2]: this is not the app log dir, it is the data log dir (i.e. used by zk to store uncommitted / replicated changes) the log file location is controlled by using the ZOO_LOG_DIR env variable which is used in the various scripts under its /bin directory

**Pull request merged**